server/utilities/encrypt.js

```diff
@@ -0,0 +1,15 @@
+const bcrypt = require('bcrypt')
```

Comment:
Looks good, but let's move this to a directory called `server/lib/` to make it more obvious that it is coming from a 3rd party.